### PR TITLE
feat-Image

### PR DESCRIPTION
**Pull request by iamsaurav28**
image (2)

### CODE REVIEW

File: Stylesheets/components/image.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  height: auto;
+}
+.img-round {
+  width: 10rem;
+  height: 10rem;
+  border-radius: 50%;
+  margin: auto;
+}
+
+.bor {
```
Comment (ttahm3d) [inline Stylesheets/components/image.css]: The classname used here could have been made more easy to recongnizable. Use something like `img-square`.

**Review by ttahm3d — APPROVED** [review]
Have left a comment, Looks good to merge for now.

File: Stylesheets/components/image.css (hunk 1 above)
Comment (iamsaurav28) [reply Stylesheets/components/image.css]: done as u suggested

**Review by Sameony — COMMENTED** [review]
Great clean code, the class names are very descriptive. You can merge it.

**Review by Sameony — APPROVED** [review]

**Pull request merged**